docs/guides/setup/setup_overview.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Setup: Overview"
+---
+# [![Setup Icon](../../images/universal_map/Gear-active.png)](setup_overview.md) Setup: Overview 
+
+import UniversalMap from '/docs/images/universal_map/_universal_map.mdx';
+
+<!--Use 'inactive' or 'active' to indicate which Universal Map steps this term has a use case within.-->
+
+<UniversalMap setup='active' connect='inactive' create='inactive' validate='inactive'/>
+
+<!-- Only keep one of the 'To best understand this document' lines.  For processes like the Universal Map steps, use the first one.  For processes like the Architecture Reviews, use the second one. -->
+
+:::note Prerequisites
+- To best understand this document, you must have completed Step 1: Setup of the [Getting Started tutorial](../../tutorials/getting_started/initialize_a_data_context.md).
```

Comment:
This feels a bit wordy, IMO. Potential alternatives:
"This document assume that you have already..."
"We recommend completing Step 1: Setup"

Also, I would have thought that we want people to complete the whole tutorial before coming to this page. No?

Reply:
Ideally, yes.  But it's also possible that people will skip the tutorial and come straight to this section, or (hopefully the more likely scenario) they may click a Technical Term for more information, then follow the "Setup" link back to this page.  In which case having a link at the top of this page back to the step in the tutorial that they were on would be useful.

Also, in general I want our prerequisites to not contain anything extraneous.  If you don't actually need to have done something to follow a guide or understand a documentation page, I don't want us to be telling people they do.  We want them to have completed the tutorial before they follow any how to guides so that they have the additional context, yes.  But if they just want to read about a given subject without doing anything (as they would on the overview or core concept/definitions pages -- which I don't intend to put prerequisites on at all) I don't think we need to require that.  If you disagree with my reasoning for these non-instruction pages, please let me know so we can evaluate and decide.

I will change the prerequisite note to "Completing Step 1: Setup in the Getting Started tutorial is recommended."